models/brats_mri_segmentation/configs/metadata.json

```diff
@@ -45,13 +46,13 @@
                     "8*n"
                 ],
                 "dtype": "float32",
-                "value_range": [
-                    0,
-                    1
-                ],
+                "value_range": [],
```

Comment:
Hi @ericspod , here I wish to ask a question about the `value_range`. Is it based on the range of the original input, or based on the range after the transforms?